README.adoc

```diff
@@ -43,72 +64,30 @@ Information I might need to debug or extend the normal usage of AeroGear softwar
-. Test your changes xref:#using-antora[using Antora]
-. Determine which functionality is this documentation related to?
-+
-* Push
-* Metrics
-* Common to all mobile services
-* Common to all services or SDKs but with some information that is specific to each.
-+
-With this information, you should be able to determine where the file should be located in the repo, eg modules/ROOT/push
-+
-If you intend to reuse a part of a file, create a partial in modules/ROOT/_partials and refer to it using an `include`
-
-If you create a link, use the following whenever possible:
-
-* getting-started.adoc
-* con_services.adoc
-* device-security.adoc
-* getting-started.adoc
-* identity-management.adoc
-* mobile-cicd.adoc
-* mobile-cli.adoc
-* mobile-clients.adoc
-* mobile-metrics.adoc
-* push-notifications.adoc
-* ref_api.adoc
-* runtime-connector.adoc
-* showcase-apps.adoc
- 
- 
-
-
-. Stick to present tense, eg `When you add the dashboard, it appears on ...`, not `When you add the dashboard, it will appear on ...`
 
+== AsciiDoc text document format
 
-== Asciidoc
+"AsciiDoc is a text document format for writing notes, documentation, articles, books, ebooks, slideshows, web pages, man pages and blogs."
 
-For a fuller list of asciidoc syntax, see link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[the quick reference], note the following syntax to avoid:
+See link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc syntax quick reference^]:
 
-* Headings using --, ~~, ^^, ++ syntax
-* Markdown
-* Source block without syntax. Try to use syntax highlighting where possible, for example `[source, bash]`
+* Avoid using headings with `--, ~~, ^^, ++`
+* Avoid using Markdown
+* Try to use syntax highlighting wherever possible, for example `[source, bash]`
 
-[NOTE]
---
-This page acts as example of documentation best practises:
+== Modular Documentation Templates
 
-* introducing terms
-* a single  procedure
-* reference material
---
-
-== Modular Docs
-
-For more information and some useful templates, see link:https://redhat-documentation.github.io/modular-docs/[Modular Docs]
+See link:https://redhat-documentation.github.io/modular-docs/[Modular Docs].
 
 [[using-antora]]
-== Using Antora to convert asciidoc to html
+== Using Antora to convert AsciiCoc to HTML
 
-NOTE: You can avoid using antora by building the docs online as described in https://github.com/finp/mobile-netlify
+NOTE: You can avoid using Antora to preview the docs with online tool as described in https://github.com/finp/mobile-netlify
 
-After modifying documentation, you can validate your changes and preview the results by:
+After modifying documentation, validate your changes and preview the results by:
 
-. Install Antora globally as described in their https://docs.antora.org/antora/1.0/install/install-antora/[install guide].
+. Installing Antora globally as described in their https://docs.antora.org/antora/1.0/install/install-antora/[install guide].
```

Comment:
it's a list, gotta be an imperative, ie Install

Reply:
resolved